cmd/istioctl/inject.go

```diff
@@ -105,6 +105,9 @@ kubectl get deployment -o yaml | istioctl kube-inject -f - | kubectl apply -f -
 			if versionStr == "" {
 				versionStr = version.Line()
 			}
+			if tag == "" {
+				tag = versionStr
```

Comment:
We don't have a buildstamp for the release version 0.2.0. Best that can be done is to use SHA, which is version.Info.GitRevision or version.Info.Version. Take a look at bin/get_workspace_status script.